lib/fastlane_core/cert_checker.rb

```diff
@@ -10,6 +10,20 @@ def self.installed?(path)
       return ids.include? finger_print
     end
 
+    def self.wwdr_certificate_installed?
+      certificate_name = "Apple Worldwide Developer Relations Certification Authority"
+      `security find-certificate -c #{certificate_name}`
```

Comment:
This looks in the current security search list, while the import command directly specifies the `login.keychain`. This causes issues when the search list was changed to a custom keychain, see https://github.com/fastlane/sigh/issues/264 .